### PR TITLE
Update and rename .github/workflows/l10n.yml to .github/Arbeitsabläuf…

### PR DESCRIPTION
**Pull request by timscharfenort33**
…e/l10n.yml

Thanks for taking the time to contribute to Git! Please be advised that the
Git community does not use github.com for their contributions. Instead, we use
a mailing list (git@vger.kernel.org) for code submissions, code reviews, and
bug reports. Nevertheless, you can use GitGitGadget (https://gitgitgadget.github.io/)
to conveniently send your Pull Requests commits to our mailing list.

Please read the "guidelines for contributing" linked above!

### CODE REVIEW

**Review by timscharfenort33 — COMMENTED** [review]
RuckZuck

File: .github/Arbeitsabläufe/l10n.yml [hunk 1]
```diff
@@ -101,5 +101,5 @@ jobs:
             ```
 
             ${{ env.COMMENT_BODY }}
-
+ 
             ```
```
Comment (timscharfenort33) [inline .github/Arbeitsabläufe/l10n.yml:105]: Good

**Review by timscharfenort33 — COMMENTED** [review]
Zur

**Comment by dscho** [comment]
@timscharfenort33 Please use your own fork for practicing how to use GitHub.

**Pull request closed without merging**